QMigrate/lut/lut.py

```diff
@@ -392,11 +419,11 @@ def __str__(self):
 
         return out
 
-    def ttimes(self, sampling_rate):
+    def serve_traveltimes(self, sampling_rate):
```

Comment:
Prefer `serve` to `get`, as it's more than just a request for an attribute, but can also be more specific (station/phase combinations) and also performs some processing rather than just spitting out a dict.